modules/phase_field/doc/content/source/kernels/ADAllenCahn.md

```diff
@@ -0,0 +1,29 @@
+# ADAllenCahn
+
+!syntax description /Kernels/ADAllenCahn<RESIDUAL>
+
+Implements the term
+
+\begin{equation}
+L(\eta,a,b,\dots)\frac{\delta F}{\delta\eta} = L(\eta,a,b,\dots)\frac{\partial f(\eta,a,b,\dots)}{\partial\eta}
```

Comment:
I would not include $F$ and its variational derivative in this description, just start with the partial of $f$. I know that below you state you are assuming that $F$ (the total free energy of the system) does not depend on the gradient. But in most cases it does so the reader is going to be confused (wondering what happened to the gradient term) until they get to the very end.

Reply:
done